masschallenge/impact-api
# AC-7697 - installing waitress and increasing timeouts for nginx

This PR swaps out gunicorn for waitress as the wsgi server used to serve impact. This is to resolve a bad issue that happens when salesforce does its daily pull of user updates. with gunicorn, a long running request takes causes the entire server to go unresponsive and return 503s.

to test:
- checkout this branch and re-run "make build" in impact.
- re-start servers via "make run-all-servers" and confirm that the impact api is accessible and works as expected. There are no out-facing functional changes
